docs/browsers-api/index.md

```diff
@@ -2,7 +2,48 @@
 sidebar_label: API
 ---
 
-# API Reference
+# @puppeteer/browsers
+
+Manage and launch browsers/drivers from a CLI or programmatically.
+
+## CLI
+
+Use `npx` to run the CLI without installing:
+
+```sh
+npx @puppeteer/browsers --help
+```
+
+or install the package as a dependency and run it from your `package.json` script:
+
+```sh
+npm i @puppeteer/browsers
+```
+
+```json
+{
+  "scripts": {
+    "browsers": "@puppeteer/browsers --help"
```

Comment:
I think if the command has `--help`  it will never run tried it with `npx @puppeteer/browsers install chrome --help`. Also when installed it seems to need the file that is created in `.bin` rather then the package name.
```suggestion
    "browsers": "browsers"
```

Reply:
Good point, I would go for `npx @puppeteer/browsers` in this case